README.md

```diff
@@ -29,42 +30,48 @@ While the following requirements are meant to be for production, the charm can b
 To get a description of all config options available, please refer to the [`config.yaml`](https://github.com/canonical/zookeeper-k8s-operator/blob/main/config.yaml) file.
 
 Options can be changed by using the `juju config` command:
+
 ```shell
 juju config zookeeper-k8s <config_option_1>=<value> [<config_option_2>=<value>]
 ```
+
 ## Usage
-### Basic usage
 
-The ZooKeeper operator may be deployed using the Juju command line as follows:
+The Apache ZooKeeper charmed operator may be deployed using the Juju command line as follows:
```

Comment:
should we call this `Apache ZooKeeper charmed operator for K8s` to have a different name than the one in VM?

Reply:
The `Apache ZooKeeper charmed operator for K8s` is a bit too long.
1. After discussing that terminology with Teodora, I can say that `charmed operator` should be replaced by just `charm`. It's much shorter, and it is consistent with the [Juju documentation](https://documentation.ubuntu.com/juju/latest/user/reference/charm/) terminology. I know the operator word adds clarity here, but it's not our call how to call it in the Juju universe. 
2. I'd put K8s a bit earlier and drop the Adposition.
Hence, I suggest the `Apache ZooKeeper K8s charm`.

Reply:
Yes, that's fine. I just meant to keep it consistent throughout